contrib/babelfishpg_tsql/src/extendedproperty.h

```diff
@@ -0,0 +1,21 @@
+#ifndef EXTENDEDPROPERTY_H
+#define EXTENDEDPROPERTY_H
+
+extern void delete_extended_property(int nkeys,
+                                     int16 db_id,
+                                     const char *schema_name,
+                                     const char *major_name,
+                                     const char *minor_name,
+                                     const char *type);
+extern void update_extended_property(int nkeys,
+                                     int16 db_id,
+                                     const char *schema_name,
+                                     const char *major_name,
+                                     const char *minor_name,
+                                     const char *type,
+                                     int attnum,
+                                     const char *new_value);
+extern void babelfish_exec_extendedproperty(PG_FUNCTION_ARGS,
```

Comment:
I am not familiar with this coding pattern. What are the considerations behind it? 
My personal preference would just be putting everything in its own function and extract common functionalities as util functions. Would like to hear your thoughts

Reply:
Yes, the common functionalities are so huge that most code are common.